### ISSUE TITLE
Clarification: Short open tag

### ISSUE THREAD

**Issue by ztl8702** [peachpiecompiler/peachpie#336]
## TL;DR
I wonder if PHP short open tags are supported, or if there is a flag that can be switched on. If not, should it be specified in the docs?

## Long version
I have an old PHP codebase that has a lot of these:
```php
<?
   // PHP code here
?>
```

mixed with the regular `<?php ?>` tags, and short echo form `<?= ?>`.

and they seem to cause a lot of `error PHP2014: Syntax error: unexpected token ''` errors. Because the Peachpie compiler only gives a "Syntax error" at the end of the file, I wasn't sure what was the exact syntax error. After a few trial and error, I found out that `<? ?>` is responsible for some of them.

Maybe a good chance to improve docs and compiler error messages?
